package.json

```diff
@@ -79,6 +79,8 @@
         "@datorama/akita-ngdevtools": "^1.0.2",
         "@types/jasmine": "~2.8.8",
         "@types/jasminewd2": "~2.0.3",
+        "@types/jest": "^25.1.4",
+        "@types/mocha": "^7.0.2",
```

Comment:
why do you need jest and mocha?

Reply:
because we use jest and mocha in tests

Reply:
No, i don't think we ever used mocha and jest for angular. karma, jasmine etc. if my memory serve me right.

Reply:
yeah this is a hack we have to find the underlying issue with tsconfig not finding types from jasmine